### DISCUSSION TITLE
The mod seems to be broken -> It's causing last exceptions

### DISCUSSION THREAD

**Discussion by ZockerGirl25703** [TitanNano/TS4ControlAnySim#78] [Q&A] [answered]
Hey there, I hope you're still active here on github. 
No idea when it broke, but your mod is causing last exceptions. When I tried to follow my sim to school I got kicked into world menu with a last exception. Twistedmexii's ''Better Exceptions'' mod says your mod is causing it. I didn't control any sim that wasn't part of my household to that time. 
![image](https://github.com/TitanNano/TS4ControlAnySim/assets/92547218/00504c48-bade-4373-a4a9-75ea766e96f5)
The control any sim mod never was mentioned in any report before. 
I also heard other people having issues with the mod since the for rent patches released.
please try to check modthesims a little more often, most people write down their issues there.
Best wishes!

**Comment by ZockerGirl25703** [comment]
[lastException.txt](https://github.com/TitanNano/TS4ControlAnySim/files/14888499/lastException.txt)
here's the last exception too

**Reply by TitanNano** [reply, marked as answer]
It appears to be unlikely that this exception has anything to do with the mod.
